charts/fluent-bit/templates/networkpolicy.yaml

```diff
@@ -0,0 +1,23 @@
+{{- if .Values.networkPolicy.enabled }}
+apiVersion: "networking.k8s.io/v1"
+kind: "NetworkPolicy"
+metadata:
+  name: {{ include "fluent-bit.fullname" . | quote }}
+  namespace: {{ .Release.Namespace | quote }}
```

Comment:
```suggestion
```

namespace is not specified for other resources, I think it can be removed here as well for consistency

Reply:
You right, I removed the line